crawler/src/main/java/com/movie/recommender/crawler/dataimport/MovieDataLoader.java

```diff
@@ -0,0 +1,67 @@
+package com.movie.recommender.crawler.dataimport;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.movie.recommender.common.model.movie.MovieCredits.CastMember;
+import com.movie.recommender.common.model.movie.MovieCredits.CrewMember;
+import com.movie.recommender.common.model.movie.MovieKeywords.Keyword;
+import com.movie.recommender.common.model.movie.MovieDetails;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MovieDataLoader {
+    private static final Logger logger = LoggerFactory.getLogger(MovieDataLoader.class);
```

Comment:
again, we use `@Slf4j` annotations in other parts of the project
keep everything consistent